Source/EasyNetQ.Management.Client.ApprovalTests/EasyNetQ.Management.Client.received.txt

```diff
@@ -0,0 +1,1093 @@
+namespace EasyNetQ.Management.Client

```

Comment:
There is no need to commit *.received.txt files. Could you please remove it?

Reply:
Done